tests/integration/browser__switch_to__alert_test.py

```diff
@@ -19,16 +19,16 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
-import os
-
 from selenium.common.exceptions import NoAlertPresentException
 
-start_page = 'file://' + os.path.abspath(os.path.dirname(__file__)) + '/../resources/start_page.html'
+from tests.integration.helpers.givenpage import GivenPage
 
 
 def test_can_accept_alert(session_browser):
-    session_browser.open(start_page)
+    GivenPage(session_browser.driver).opened_with_body("""
+        <p>
+        <input id="alert_btn" type="button" onclick="alert('Добрый день')" value="Запустить">
```

Comment:
I guess here we are not forced to test unicode... So let's stick to English texts...

Reply:
Agreed. Done it in 26528b1652d2f9192747d0cc72622707cdfb9d4f